### ISSUE TITLE
TypeScript language service died 5 times

### ISSUE THREAD

**Issue by ReidBraswell** [microsoft/vscode#85608]
Version: 1.40.2 (user setup)
Commit: f359dd69833dd8800b54d458f6d37ab7c78df520
Date: 2019-11-25T14:54:45.096Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.16299

Steps to Reproduce:

1. Open VS Code
2. Open a `.ts` or `.tsx` file
3. Wait until you receive the following notification

![image](https://user-images.githubusercontent.com/318944/69644729-06b31e80-1033-11ea-81ba-02edf94a2b46.png)

Does this issue occur when all extensions are disabled?: **No**

**Comment by mjbvz** [comment]
So this doesn't happen when you disable all installed extensions? What extension triggers this?

Also, does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/)?

**Comment by ReidBraswell** [comment]
@mjbvz thanks for the quick response! I could not reproduce the TypeScript language service issue with when running `code --disable-extensions` in the same project. I'm not sure which extension was triggering it thought. I updated to `v1.40.2` this morning and it started happening after that update. Previously on `v1.40.1` it was not occurring, and I had the same list of extensions enabled.

I downloaded the latest VS Code insiders build (`v3.7.3-insiders.20191123`) and the issue is not occurring.
